03_launch_mgmt_cluster.sh

```diff
@@ -147,7 +147,8 @@ EOF
   fi
 
   # Copy the generated configmap for ironic deployment
-  cp "$IRONIC_DATA_DIR/ironic_bmo_configmap.env"  "${BMOPATH}/ironic-deployment/keepalived/ironic_bmo_configmap.env"
+  # We always use the keepalived component so this is where we put the configmap
```

Comment:
Cannot we use default folder that will be the base of all the other instead of an independent component like keepalived or tls ..

Reply:
No we cannot use the default as base for the other, because it has a different ConfigMap than the keepalived component. ConfigMaps cannot be easily merged, instead we replace it completely. So we use the [base](https://github.com/Nordix/baremetal-operator/tree/lentzi90/ironic-kustomize-components/ironic-deployment/base) as base instead since it has no ConfigMap at all.

In the future I would like to change this so that we create a temporary overlay in metal3-dev-env instead of messing with BMO, similar to what the [deploy.sh](https://github.com/Nordix/baremetal-operator/blob/lentzi90/ironic-kustomize-components/tools/deploy.sh#L101-L136) script is doing in my PR). But I think it makes sense to take this in first as it is a smaller change.

Reply:
ok 
/lgtm